src/404.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: base_layout
+title: "404: Page not found"
```

Comment:
So after taking a look at things, I think 11ty by default outputs `<folder>/file.html` so to override add the following line please:

```suggestion
title: "404: Page not found"
permalink: 404.html
```
I would also suggest to add `_site` to the `clean` command, as this change will require removing that folder (otherwise it will just overwrite and leave the old 404 folder).

And then you can try running docker to check it's working 100%
```bash
DOCKER_BUILDKIT=1 docker build --tag mir-server . 
docker run -ti -p 8045:80 mir-server
```

Reply:
Thanks @carkod, `_site/` is already removed by the clean command.